Boeing-Ontology/ontology/Boeing.sadl

```diff
@@ -68,20 +68,58 @@ SoftwareCoding is a type of CODE_DEVELOPMENT.
 // Support for characterizing message types, valid signals, and tests
 // that verify these correspondences
 
-Message is a type of DATA_DICTIONARY_TERM.
+Message is a type of INTERFACE.
 
 Signal is a type of DATA_DICTIONARY_TERM.
-    enumValue describes Signal with values of type string.
-
-MessageSignal is a type of DATA_DICTIONARY_TERM.
-    message describes MessageSignal with a single value of type Message.
-    signal describes MessageSignal with a single value of type Signal.
-
-IDD is a type of INTERFACE.
-    messageSignal describes IDD with values of type MessageSignal.
-
-IDD_Test is a type of TEST.
-    verifies of IDD_Test only has values of type MessageSignal.
+    message describes Signal with a single value of type Message.
+    
+// content of IDD_Doc can be Message and other things
+IDD_Doc is a type of DOCUMENT.
+
+// subclass from core ontology related to SBVT and IDD
+SBVT_Test_Procedure is a type of TEST_PROCEDURE. 
```

Comment:
This PR does have a TEST--verifies-->ENTITY association at present, so that
concern is covered.

A TEST_STEP is really just the identification of a TEST, along with any
subsequent TEST_STEPs, so the TEST_STEP doesn't add any verification that isn't
already covered by the TEST it identifies and therefore shouldn't need a
"verifies" property.

The TEST_PROCEDURE has no additional context other than being a COLLECTION of
initial TEST_STEP entries, so the only additional verification claim it could
make would be relative to the aggregation of the TESTs.  However, adding a
"verifies" property at the TEST_PROCEDURE level raises a concern of coherence: if
a TEST_PROCEDURE were to indicate that it verifies a high-level ENTITY *but* the
set of TEST entries underneath that TEST_PROCEDURE are not an exact correlation
to all of the sub-ENTITY elements of that ENTITY, which one is correct (the
TEST_PROCEDURE.verifies or the individual TEST.verifies)?  This also affects
extensibility: if a TA1 determines that an additional test is needed to validate
the higher-level construct, then the TEST_PROCEDURE can no longer be said to
verify that construct.  In this consideration, the intent is that in our ontology
the TEST object is the atomic unit of verification and that any implication that
higher-level elements are fully verified is a TA3 evaluation/conclusion that we
don't have enough information to supply or synthesize.

In general, the heirarchy of TEST_PROCEDURE::TEST_STEP::TEST is concerned with
describing the relationship of each TEST to any other TEST, rather than an
association of different testing importances, but there is no constraint on the
fidelity of a particular TEST, so it should be possible to define a TEST to
associate to any desired target ENTITY to capture the desired fidelity.